prov/sm2/src/sm2_coordination.c

```diff
@@ -268,8 +268,13 @@ ssize_t sm2_file_open_or_create(struct sm2_mmap *map_shared)
 		}
 
 		common_fd = open(SM2_COORDINATION_FILE, O_RDWR);
-		if (common_fd > 0) {
```

Comment:
it can be 0? I thought fd should be positive

Reply:
lol, so did I, which is why I wrote it that way in the first place.
> The return value of open() is a file descriptor, a small,
       nonnegative integer that is an index to an entry in the process's
       table of open file descriptors.